Documentation/6.0.2/README.rst

```diff
@@ -0,0 +1,1518 @@
+
+If you want to use a remote registry for the Overcloud container images, you need to add the iptables rule on Director (Undercloud Machine) with the Undercloud IP address or interface with external connectivity for NAT. In the below example, the Undercloud IP address is 192.168.24.1 and the external interface name is eth0:
+
+::
+
+    sudo iptables -A POSTROUTING -t nat -s 192.168.24.1/24 -j MASQUERADE
+    (or)
+    sudo iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
+
+
+Phase 2: Configure the basic Overcloud.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Follow the upstream OpenStack documentation *up to the step where* the ``openstack overcloud deploy`` command is run using the CLI or starting the Overcloud deployment (starting the Overcloud creation) in the UI.
+
+These are the OpenStack instructions:
+
+    * Configuring the container registry details: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/configuring-a-container-image-source
+    * Using the CLI: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/chap-configuring_basic_overcloud_requirements_with_the_cli_tools
+    * Using the UI: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/chap-configuring_basic_overcloud_requirements_with_the_ui_tools
+
+
+
+Phase 3: Modify the Overcloud qcow image (for example, overcloud-full.qcow2) to include Nuage components.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The steps for modifying overcloud-full.qcow2 are provided in the README.md file: https://github.com/nuagenetworks/nuage-ospdirector/tree/OSPD13/image-patching/stopgap-script/README.md .
+
+
+Phase 4: Adding Nuage Heat Templates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Copy the nuage-tripleo-heat-templates folder form `nuage-tripleo-heat-templates <../../nuage-tripleo-heat-templates>`_ to `/home/stack/` directory on undercloud.
+
+.. Note:: As an end-user, it is a best practice to manage the templates using GIT. If you are aware of git workflow, you can run following command to manage your nuage-template change via GIT.
+
+    ::
+
+        cd /home/stack
+        git clone -b OSPD13 https://github.com/nuagenetworks/nuage-ospdirector nuage-ospdirector ( here nuagenetworks can be replaced with your own fork)
+        ln -s nuage-ospdirector/nuage-tripleo-heat-templates .
+
+
+Copy the roles from `/usr/share/openstack-tripleo-heat-templates/roles` to `/home/stack/nuage-tripleo-heat-templates/roles`
+
+    ::
+
+        cp /usr/share/openstack-tripleo-heat-templates/roles/* /home/stack/nuage-tripleo-heat-templates/roles/
+
+
+**For an AVRS integration please follow below steps as well**:
```

Comment:
Revise text as follows.
From:
**For an AVRS integration please follow below steps as well**
To (no bold): 
For an AVRS integration, perform the following steps

Reply:
Since it becomes tricky sometimes when you do a deployment with and without AVRS, We made it bold for the sole reason that user who want AVRS integration doesn't miss these steps.

Reply:
In tech docs, bold is not usually used to get the user's attention. Neither is using all capital letters for a word or phrase. However, I am working on a project with PLM to restructure the OSPD documentation so I am OK with this in the current document. I can spend more time later figuring out how to revise the instructions for a deployment with and without AVRS.

Reply:
done